get_training_set.py

```diff
@@ -0,0 +1,115 @@
+import xml.etree.ElementTree as xmlTree
+import numpy as np
+import os
+import tensorflow as tf
+from tqdm import tqdm
+
+class_names = ['car', 'watcher', 'base', 'armor_red', 'armor_blue']
+
+
+@tf.function
+def int64_feature(value):
+    return tf.train.Feature(int64_list=tf.train.Int64List(value=[value]))
+
+
+def int64_list_feature(value):
+    return tf.train.Feature(int64_list=tf.train.Int64List(value=value))
+
+
+def bytes_feature(value):
+    return tf.train.Feature(bytes_list=tf.train.BytesList(value=[value]))
+
+
+def bytes_list_feature(value):
+    return tf.train.Feature(bytes_list=tf.train.BytesList(value=value))
+
+
+def float_feature(value):
+    return tf.train.Feature(bytes_list=tf.train.FloatList(value=[value]))
+
+
+def float_list_feature(value):
+    return tf.train.Feature(float_list=tf.train.FloatList(value=value))
+
+
+def convert(size, box):
+    x1 = box[0] / size[0]
+    y1 = box[1] / size[1]
+    x2 = box[2] / size[0]
+    y2 = box[3] / size[1]
+    return [x1, y1, x2, y2]
+
+
+def convert_image(image_path):
+    image_string = open(image_path, 'rb').read()
+    return image_string
+
+
+def convert_annot(annot_path):
+    xml_file = open(annot_path)
+    tree = xmlTree.parse(xml_file)
+    root = tree.getroot()
+    size = root.find('size')
+    w = int(size.find('width').text)
+    h = int(size.find('height').text)
+    objt = []
+    x1 = []
+    y1 = []
+    x2 = []
+    y2 = []
```

Comment:
1. This is the style I also prefer and tried. However, I do not know why but after `tf.io.parse_single_example`, the `tf.io.parse_tensor` needs `for` loops to iterate every picture instead of reading parallelly, so the reading speed is some what low.
2. https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/using_your_own_dataset.md#conversion-script-outline-conversion-script-outline. This is the TFRecord style that I follow, and also the original project only supports this style, so I think maybe there is no need to use `tf.io.serialize_tensor`.